create-fixlet/cURL/README.md

```diff
@@ -1,3 +1,37 @@
+Equivalent Action in Console
+---
+Right-click a Fixlet and select "Create New Fixlet...".
+
+Explanation of ['fixlet.xml'](https://github.com/bigfix/restapi-examples/blob/master/create-fixlet/cURL/fixlet.xml)
```

Comment:
You can also add the link to https://github.com/bigfix/restapi-examples/pull/6/files#diff-07c900c061aeee62a76764f76c336634R38.